sdcclient/_secure.py

```diff
@@ -16,6 +16,17 @@ def __init__(self, token="", sdc_url='https://secure.sysdig.com', ssl_verify=Tru
 
         self.customer_id = None
         self.product = "SDS"
+        self._policy_v2 = None
+
+    @property
+    def policy_v2(self):
```

Comment:
Shouldn't this be implied by the fact that you instantiate `SdSecureClient` instead of `SdSecureClientV1`?

Reply:
The idea of this property is to be able to test if the provider support policy_v2 or not. With it is easier to write code that works with both versions of the policy API, something like:
```python
client = SdSecureclient()
if not client.policy_v2:
    client = SdSecureClientV1()
ok, res = client.list_policies()
...
```